libs/core/src/lib/multi-input/multi-input.component.scss

```diff
@@ -17,11 +19,21 @@
         overflow: auto;
     }
 
-    .fd-multi-input-token-spacing {
-        margin: 0 4px 4px 0;
+    .fd-multi-input-tokenizer {
+        width: 100%;
+    }
+
+    .fd-multi-input-tokenizer-input {
+        border: none !important;
```

Comment:
Do we have to use `!important` :( ?

Reply:
This css class felt code-smelly to me as well but I couldn't find another way, I was hoping to solicit help from the team on these properties

Reply:
Please do not use `!important` in library, always try to use stronger selectors